core/base/src/TROOT.cxx

```diff
@@ -2574,14 +2544,258 @@ const char**& TROOT::GetExtraInterpreterArgs() {
 }
 
 ////////////////////////////////////////////////////////////////////////////////
+
+#ifdef ROOTPREFIX
+static Bool_t IgnorePrefix() {
+   static Bool_t ignorePrefix = kFALSE;
```

Comment:
`static Bool_t ignorePrefix = gSystem->Getenv("ROOTIGNOREPREFIX");` would actually be the same plus thread-safe.